charts/partials/templates/_helpers.tpl

```diff
@@ -12,3 +12,12 @@ For example "11,22,55,44" will become "11","22","55","44"
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/* vim: set filetype=mustache: */}}
+{{/* Check if the specified CRD exists in the target cluster */}}
```

Comment:
Is the idea here to prevent rendering a CRD if that CRD already exists in the cluster? Our practice is to always render resources, even if they already exist in the cluster.  This is important for upgrades where we may apply resources that already exist in order to update them.

Also, it seems this isn't working since installing CRDs succeeds even though I have the httproute CRD already installed:

```
$ bin/linkerd install --crds --ignore-cluster --set enableGrpcRoutes=true --set enableHttpRoutes=true > /dev/null
Rendering Linkerd CRDs...
Next, run `linkerd install | kubectl apply -f -` to install the control plane.
$ k get crds | grep route                                                                                        
ingressroutes.traefik.containo.us          2023-11-21T17:17:37Z
ingressrouteudps.traefik.containo.us       2023-11-21T17:17:37Z
ingressroutetcps.traefik.containo.us       2023-11-21T17:17:37Z
httproutes.policy.linkerd.io               2024-02-19T19:45:53Z
httproutes.gateway.networking.k8s.io       2024-02-19T19:45:53Z
```

Reply:
That was the idea yes, on the assumption that because the Gateway API CRDs are aren't actually _linkerd_ CRDs that we'd want to ensure they _are_ installed but shouldn't _fail_ the install if `linkerd` wasn't the thing that installed them to begin with.

With respect to your observed behavior (just to clarify) - in a cluster where the Gateway `HTTPRoute` CRD is already installed (separate from the `linkerd-crds` install) and when installing the `linkerd-crds` it _didn't_ fail? If so, that _is_ the intended behavior of the addition.

Reply:
@adleong I've reverted the addition on further consideration. I _do_ think it's worth a think over though, as a future `TODO`.